lib/drivers/redis.js

```diff
@@ -110,6 +110,9 @@
                 data[data.length - 1] = data[data.length - 1].map(function (index) {
                     return parseInt(index, 10);
                 });
+                data.forEach(function (row, index) {
```

Comment:
It depends on the config. If you have 5 frontends that can process the host, then the most specific one with a configured backend should win... from what I can tell, that's how this actually works. In my testing anyway, the response comes out in order from most specific (`blah.liftopia.net`) to the wildcard (`*.liftopia.net`) and working its way down the domain tree to the global wildcard (`*`). I'm pretty confident this will always be in the right order, otherwise doing these kinds of searches would result in wildly unordered routing.